sdk/src/version/version.cc

```diff
@@ -9,16 +9,16 @@ namespace sdk
 namespace version
 {
 const int MAJOR_VERSION     = 1;
-const int MINOR_VERSION     = 6;
-const int PATCH_VERSION     = 1;
+const int MINOR_VERSION     = 7;
+const int PATCH_VERSION     = 0;
 const char *PRE_RELEASE     = "NONE";
 const char *BUILD_METADATA  = "NONE";
-const int COUNT_NEW_COMMITS = 27;
-const char *BRANCH          = "pre_release_1.6.1";
-const char *COMMIT_HASH     = "38012fab34faa50d5020959b68eba77ac485b508";
-const char *SHORT_VERSION   = "1.6.1";
+const int COUNT_NEW_COMMITS = 35;
+const char *BRANCH          = "pre_release_1.7.0";
+const char *COMMIT_HASH     = "dd7e257b6de71eeaf9e3149530962301705b9a0d";
+const char *SHORT_VERSION   = "1.7.0";
 const char *FULL_VERSION =
-    "1.6.0-NONE-NONE-27-pre_release_1.6.1-38012fab34faa50d5020959b68eba77ac485b508";
+    "1.7.0-NONE-NONE-35-pre_release_1.7.0-dd7e257b6de71eeaf9e3149530962301705b9a0d";
 const char *BUILD_DATE = "Thu Sep 22 18:05:21 UTC 2022";
```

Comment:
`buildscripts/pre-commit` should update the `BUILD_DATE` here?

Reply:
thanks, updated